configure.ac

```diff
@@ -576,6 +589,8 @@ if test "$CONFIG_HOST" = "Linux"; then
     KVM=1
     AC_SUBST(KVM)
   fi
+elif test "$CONFIG_HOST" = "Darwin"; then
```

Comment:
This seems to be enabling
DNATIVE and KVM.
Not the best place for this code & `elif`
won't be needed.

Reply:
I put it there because the `if test "$CONFIG_HOST" = "Linux"; then` above so it's blocks of OS-specific code. But I can move it of course.

Reply:
Yes, please move.
There is no rule to have only single
`if test "$CONFIG_HOST" = "Linux"`
and this particular one enables dnative
and kvm. If yours would enable e.g.
something else for dpmi or kvm on
darwin, then of course. But since it
does a completely unrelated thing,
better to add a new block.

Reply:
No need to move anything actually.
Just replace `elif` with an `if`.

Reply:
done